<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showPictureDialog</class>
 <widget class="QDialog" name="showPictureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
background-color: rgb(51, 153, 255);




</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>40</y>
      <width>57</width>
      <height>57</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image: url(:/re/back.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_rfid">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>167</y>
      <width>17</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/triangle.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rfid">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>160</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    border:none;
    background:transparent;
    font: 18pt &quot;黑体&quot;;
    color: rgb(248, 248, 255);
}</string>
    </property>
    <property name="text">
     <string>查看图片</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEncoder">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    border:none;
    background:transparent;
    font: 18pt &quot;黑体&quot;;
    color: rgb(248, 248, 255);
}</string>
    </property>
    <property name="text">
     <string>查看文件</string>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>0</y>
     <width>1166</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 248, 255);</string>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>56</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/left.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>56</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/right.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>1146</width>
       <height>611</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(51, 153, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>241</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>共有0张图片，当前第0张</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>0</y>
       <width>791</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pdmname">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>271</width>
       <height>35</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;</string>
     </property>
     <property name="text">
      <string>bmp,jpg</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>30</y>
       <width>301</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>设置查看图片格式(多个格式用逗号隔开)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_32">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>710</y>
       <width>54</width>
       <height>54</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/saves.bmp);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_21">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>710</y>
       <width>54</width>
       <height>54</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/cancel.bmp);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
